src/main/java/mate/jdbc/dao/impl/ManufacturerDaoImpl.java

```diff
@@ -0,0 +1,119 @@
+package mate.jdbc.dao.impl;
+
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+import mate.jdbc.dao.ManufacturerDao;
+import mate.jdbc.exception.DataProcessingException;
+import mate.jdbc.lib.Dao;
+import mate.jdbc.model.Manufacturer;
+import mate.jdbc.util.ConnectionUtil;
+
+@Dao
+public class ManufacturerDaoImpl implements ManufacturerDao {
+    @Override
+    public List<Manufacturer> getAll() {
+        List<Manufacturer> allManufacturer = new ArrayList<>();
+        String insertRequest = "SELECT * FROM manufacturers WHERE is_deleted = false;";
+        try (Connection connection = ConnectionUtil.getConnection();
+                PreparedStatement getAllManufacturersStatement =
+                        connection.prepareStatement(insertRequest)) {
+            ResultSet resultSet = getAllManufacturersStatement.executeQuery();
+            while (resultSet.next()) {
+                Manufacturer manufacturer = getManufacturer(resultSet);
+                allManufacturer.add(manufacturer);
+            }
+        } catch (SQLException e) {
+            throw new DataProcessingException("Can't get all manufacturers from DB ", e);
+        }
+        return allManufacturer;
+    }
+
+    @Override
+    public Optional<Manufacturer> get(Long id) {
+        String insertRequest = "SELECT * FROM manufacturers WHERE is_deleted = false AND id = ?";
+        try (Connection connection = ConnectionUtil.getConnection();
+                PreparedStatement getManufacturersStatement =
+                        connection.prepareStatement(insertRequest)) {
+            getManufacturersStatement.setLong(1, id);
+            ResultSet resultSet = getManufacturersStatement.executeQuery();
+            if (resultSet.next()) {
+                Manufacturer manufacturer = getManufacturer(resultSet);
+                return Optional.of(manufacturer);
+            }
+            return Optional.empty();
+        } catch (SQLException e) {
+            throw new DataProcessingException("Can't get manufacturer from DB ", e);
+        }
+    }
+
+    @Override
+    public Manufacturer create(Manufacturer manufacturer) {
+        String insertManufacturerRequest =
+                "INSERT INTO manufacturers(name, country) values(?, ?);";
```

Comment:
Also, don't forget about spaces 
```suggestion
                "INSERT INTO manufacturers (name, country) VALUES (?, ?);";
```